### PR TITLE
Task/tim dedicated page

### PR DESCRIPTION
**Pull request by cs-pratap-nikam**

### CODE REVIEW

File: dashboards/Threat Intel.json [hunk 1]
```diff
@@ -194,18 +188,18 @@
...
-                                                "alias": "typeOfFeed",
-                                                "field": "typeOfFeed.itemValue"
+                                                "alias": "type",
+                                                "field": "type.itemValue"
                                             },
                                             {
                                                 "operator": "select",
                                                 "alias": "color",
-                                                "field": "typeOfFeed.color"
+                                                "field": "type.color"
```
Comment (cs-pooja-singh) [inline dashboards/Threat Intel.json:197]: this will still be typeOfFeed

File: info.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "threat-intel-management",
+    "name": "threatIntelManagement",
```
Comment (cs-pooja-singh) [inline info.json:2]: why this name change?

File: modules/threat_intel_feeds/mmd.json [hunk 3]
```diff
@@ -271,58 +271,15 @@
...
-            "sattrib": "\/api\/3\/staging_model_metadatas\/f11d681a-58e5-4b53-adb6-085f6462593e",
-            "uuid": "f2f28a15-c59a-40e5-9e0b-4d3f656ed4e9",
-            "displayName": "{{ relatedThreatActors }}",
-            "descriptions": {
-                "singular": "Related Threat Actors"
-            }
-        },
-        {
-            "@id": "\/api\/3\/attribute_metadatas\/56a576df-37c9-4f10-9b99-45113734464a",
+            "@id": "\/api\/3\/attribute_metadatas\/6e8c01e9-d4bb-4eac-9435-1037a48025d9",
```
Comment (cs-pooja-singh) [inline modules/threat_intel_feeds/mmd.json:282]: don't remove actor mapping please? 
what are the changes needed in MMD?

File: playbooks/TIM Automation/collection.metadata.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
     "@context": "\/api\/3\/contexts\/WorkflowCollection",
     "@type": "WorkflowCollection",
-    "name": "Workspace Automation",
+    "name": "TIM Automation",
```
Comment (cs-pooja-singh) [inline playbooks/TIM Automation/collection.metadata.json:4]: why is the report file below deleted?

File: playbooks/globalVariables.json [hunk 5]
```diff
@@ -1 +1,8 @@
-[]
\ No newline at end of file
+[
+    {
+        "id": 11,
+        "name": "Indicator_Feed_Reputation_Preference",
```
Comment (cs-pooja-singh) [inline playbooks/globalVariables.json:4]: is this checked in twice?

File: picklists/ThreatIntelFeedType.json [hunk 6]
```diff
@@ -14,7 +14,6 @@
             "icon": null,
```
Comment (cs-pooja-singh) [inline picklists/ThreatIntelFeedType.json:14]: there is an item added to Tasks Types picklist also. please include that

File: dashboards/Threat Intel.json (hunk 1 above)
Comment (cs-pratap-nikam) [reply dashboards/Threat Intel.json]: this dashboard no longer needed. so removed it.

File: info.json (hunk 2 above)
Comment (cs-pratap-nikam) [reply info.json]: reverted this changes.

File: playbooks/TIM Automation/collection.metadata.json (hunk 4 above)
Comment (cs-pratap-nikam) [reply playbooks/TIM Automation/collection.metadata.json:4]: Its not deleted, it moved in another folder.

File: picklists/ThreatIntelFeedType.json (hunk 6 above)
Comment (cs-pratap-nikam) [reply picklists/ThreatIntelFeedType.json:14]: Included TaskType picklist

**Review by cs-pooja-singh — APPROVED** [review]

**Pull request merged**